global.json

```diff
@@ -1,16 +1,16 @@
 {
   "tools": {
-    "dotnet": "6.0.100-preview.1.21103.13",
```

Comment:
The missing `,` here is causing problems when Maestro attempts to update this PR further. I don't have permission to push changes to this branch directly or I would've just fixed it. 

https://github.com/dotnet/core-eng/issues/13028